module/kvmfr.c

```diff
@@ -655,6 +655,8 @@ MODULE_LICENSE("GPL v2");
 MODULE_AUTHOR("Geoffrey McRae <geoff@hostfission.com>");
 MODULE_AUTHOR("Guanzhong Chen <quantum2048@gmail.com>");
 MODULE_VERSION("0.0.9");
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,16,0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,13,0)
```

Comment:
This #if is superfluous as MODULE_IMPORT_NS supports string since 5.10

Ref: https://github.com/torvalds/linux/commit/33def8498fdde180023444b08e12b72a9efed41d

Reply:
Then I close this in favour of #1149.